scripts/ci/distcheck.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -e
+
+# CC LD AR CXX has to be predifubed
+#
+
+export PKG_CONFIG_PATH="$HOME/cunit-install/x86_64/lib/pkgconfig:${PKG_CONFIG_PATH}"
```

Comment:
Ideally this should go to external environment setup. Also if you are building docker images, it might be easier to just install CUnit into system path properly (by installing proper packages).

Reply:
yes, or that can be in Dockerfile or ubuntu18 already has version which we need. I will re check.